src/utilities/positionRelative.ts

```diff
@@ -0,0 +1,46 @@
+import { MaybeRefOrGetter, toValue } from 'vue'
+import { PositionMethod, Position, MousePosition } from '@/types/position'
+
+export const lowerRight = function(position: MaybeRefOrGetter<MousePosition>): PositionMethod {
+  return function(): Position {
+    const { x, y } = toValue(position)
+
+    return {
+      top: y,
+      left: x,
+    }
+  }
+}
```

Comment:
Could these be made more generic and just use the built in placement methods we already have but override the target to mouse position?

Reply:
Ok what do you think about now?

Reply:
Nice, beautiful